tensorflow/core/platform/windows/intrinsics_port.h

```diff
@@ -24,6 +24,9 @@ limitations under the License.
 #include "tensorflow/core/platform/types.h"
 
 #define _mm_load_pd1 _mm_load1_pd
+
+// only define these intrinsics if imminitrin.h by Intel wasn't included
+#ifndef _INCLUDED_IMM
```

Comment:
I think this will break vs2015 builds if compiled with /arch:avx2.
maybe instead of 
#ifndef _INCLUDED_IMM
use
#if _MSC_VER < 1910
to preserve vs2015 since this is what is out there.

CI will not see this because it does not build for avx2 but out does.